.github/ISSUE_TEMPLATE.md

```diff
@@ -1,34 +1,17 @@
-Is this a question, feature request, or bug report?
-
-**QUESTION**
-
-Have you checked our documentation at https://cockroachlabs.com/docs/stable/? If you could not find an answer there, please consider asking your question in our community forum at https://forum.cockroachlabs.com/, as it would benefit other members of our community.
-
-Prefer live chat? Message our engineers on our Gitter channel at https://gitter.im/cockroachdb/cockroach.
-
-**FEATURE REQUEST**
-
-1. Does an issue already exist addressing this request? If yes, please add a :+1: reaction to the existing issue. If not, move on to step 2.
-
-2. Please describe the feature you are requesting, as well as your proposed use case for this feature.
-
-3. Indicate the importance of this issue to you (blocker, must-have, should-have, nice-to-have). Are you currently using any workarounds to address this issue?
+Is this a bug report or a feature request?
 
 **BUG REPORT**
 
-1. Please supply the header (i.e. the first few lines) of your most recent
```

Comment:
Seems useful to have a basic set of info to ask for to avoid an unnecessary round trip.